public/javascripts/controller/data.js

```diff
@@ -23,6 +23,23 @@ angular.module('undp-ghg-v2')
       }
       $scope.sidebarPartial('notes');
 
+      // replace existing record with new record and remove conflict
+      $scope.saveNewRecord = function() {
+        $scope.selectedRow = $scope.selectedRow.conflict;
+      }
+
+      // keep existing record and remove conflict
+      $scope.keepOldRecord = function() {
```

Comment:
So the application should warn the user when an overwrite will happen - however if the user proceeds then this will replace the record with the new record.  It should not revert. @gilgad

Reply:
forget what i said... I just saw how the code actually works.... and it works better than i had initially described.